scripts/grid.js

```diff
@@ -0,0 +1,386 @@
+import {
```

Comment:
Does this file not have to have a `.mjs` extension? If you load a script asset via the engine's asset system, it make the assumption a `.js` is non-ESM and `.mjs` is ESM. There are separate code paths. cc @marklundin